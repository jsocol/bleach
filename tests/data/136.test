<div id="64"><!DOCTYPE x[<!ENTITY x SYSTEM "http://html5sec.org/test.xxe">]><y>&x;</y>//["'`-->]]>]</div>
