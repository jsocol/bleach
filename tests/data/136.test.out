<div id="64">]&gt;&amp;x;//["'`--&gt;]]&gt;]</div>
